content/docs/tech-resources/webhook-event-reference.md

```diff
@@ -12,26 +12,12 @@ Additional resources: [Webhook Authentication](https://www.sparkpost.com/docs/te
 
 ## Event Types
 
-Events which are sent from SparkPost webhooks are organized into different event types. Below is a table that provides you with a list of these event types and map containing each event which fall into their respective event types.
-
-<div id="content-event-types" style="margin: 0 1.5rem"></div>
-<script id="event-type-template" type="text/x-handlebars-template"><table id="event-types"> <thead> <tr> <td>Event Type</td> <td>JSON Key</td> <td>Description</td> <td>Events</td> </tr> </thead> <tbody> {{#each this}} <tr> <td>{{ name }}</td> <td>{{ key }}</td> <td>{{ description }}</td> <td>{{ events }}</td> </tr> {{/each}} </tbody> </table></script>
+Events which are sent from SparkPost webhooks are organized into different event types. Below is a table that provides you with a [list of these event types](https://developers.sparkpost.com/api/webhooks/#header-webhook-event-types) and map containing each event which fall into their respective event types.
```

Comment:
why did we change this ?

Reply:
ohh I found the backstory. https://messagebird.slack.com/archives/C02PVLZGF47/p1650640315695029?thread_ts=1650637243.965329&cid=C02PVLZGF47

Reply:
Sorry, forgot to mention it 🤦‍♂️